public/js/pimcore/object/classes/data/multiselect.js

```diff
@@ -99,13 +103,147 @@ pimcore.object.classes.data.multiselect = Class.create(pimcore.object.classes.da
             return stylingItems;
         }
 
-        const valueStore = new Ext.data.JsonStore({
+        const optionsStore = new Ext.data.JsonStore({
             fields: ["key", {name: "value", allowBlank: false}],
             data: datax.options
         });
+        let optionsGrid = this.createoptionsgrid(optionsStore);
+        this.optionsModel = optionsGrid.getSelectionModel();
+
+        const defaultValueStore = new Ext.data.JsonStore({
+            fields: ["value"],
+            data: datax.defaultValue
+        });
+        let defaultValueGrid = this.createdefaultvaluegrid(defaultValueStore);
+        this.defaultValueModel = defaultValueGrid.getSelectionModel();
+
+        let appendedStylingItems = stylingItems.concat([
+            {
+                xtype: "numberfield",
+                fieldLabel: t("maximum_items"),
+                name: "maxItems",
+                value: datax.maxItems,
+                minValue: 0
+            },
+            {
+                xtype: "combo",
+                fieldLabel: t("multiselect_render_type"),
+                name: "renderType",
+                itemId: "renderType",
+                mode: 'local',
+                store: [
+                    ['list', 'List'],
+                    ['tags', 'Tags']
+                ],
+                value: datax["renderType"] ? datax["renderType"] : 'list',
+                triggerAction: "all",
+                editable: false,
+                forceSelection: true
+            },
+            {
+                xtype: "textfield",
+                fieldLabel: t("options_provider_class"),
+                width: 600,
+                name: "optionsProviderClass",
+                value: datax.optionsProviderClass
+            },
+            {
+                xtype: "textfield",
+                fieldLabel: t("options_provider_data"),
+                width: 600,
+                value: datax.optionsProviderData,
+                name: "optionsProviderData"
+            },
+            {
+                xtype: "textfield",
```

Comment:
```suggestion
                xtype: "hidden",
```
to avoid to show 
![image](https://github.com/user-attachments/assets/196aa848-3c2f-43dc-bdc7-d2ef204c28ec)